chore(cryptarchia-consensus): Add time relay to consensus relays

strinnityk opened the pull request:
## 1. What does this PR implement?
- Remove leftover comments from https://github.com/logos-co/nomos/pull/1183
- Add time relay to `CryptarchiaConsensusRelays`

## 2. Does the code have enough context to be clearly understood?
Yes

## 3. Who are the specification authors and who is accountable for this PR?
@AlejandroCabeza 

## 4. Is the specification accurate and complete?
N/A

## 5. Does the implementation introduce changes in the specification?
N/A

## Checklist

> [!WARNING]  
> Do not merge the PR if any of the following is missing:

* [X] 1. Description added.
* [X] 2. Context and links to Specification document(s) added.
* [X] 3. Main contact(s) (developers and specification authors) added
* [X] 4. Implementation and Specification are 100% in sync including changes. This is critical.
* [X] 5. Link PR to a specific milestone.

ntn-x2 commented:
These changes are also part of my PR. I think it would be best not to merge this to avoid any conflicts, especially for the cryptarchia stuff...

danielSanchezQ commented:
> These changes are also part of my PR. I think it would be best not to merge this to avoid any conflicts, especially for the cryptarchia stuff...

Yeah, this make sense. It can be stalled for now and after we merge the big stuff we do this.

danielSanchezQ commented:
@ntn-x2 this was already took care on your PR right? If so we can close this one.

strinnityk commented:
> I haven't checked how Alex has updated the broken services, specifically the cryptarchia one. So perhaps @AlejandroCabeza could compare what I did with what he did, and perhaps update the PR to change the approach, if his is a better one. I limited myself to move things around to make it compile.

Well, it seems you covered everything so nothing to add there 😂
I just removed some leftover comments from stuff you removed in the other PR and added the `TimeService` relay to the `CryptarchiaConsensusRelays`.

ntn-x2 reviewed (approved):
Amazing!
